### PR TITLE
Update user profile fields

### LINKED ISSUE #731: Atualizar perfil do usuário
## Feature Description

- Atualizar campos do cadastro de organizações no modal de criação de organizações de acordo com as inclusões dos novos campos: 
  * E-mail
  * Telefone
  * Site
  * Logomarca
  * Tipo: Pública, Privada ou Mista
  * Categoria: Universidade, Instituto, Associação, Fundação, Cooperativa, Empresa, Outro 

- Alterar o label do campo Instituição para Instituição/Empresa e o Local place para "Nome da sua instituição ou empresa"
- Checar verificação no campo Telefone, que aceita apenas formato de telefone celular. Ou seja, telefones fixos com DDD + 8 digitos não é aceito.
-  Campos CPF, CEP e Telefone aparecem erro de invalidPattern quando editamos qualquer outro campo.
---------------

## Acceptance criteria

* 

## Implementation Brief

*

### PR DESCRIPTION
**Pull request by mateus4k**
## Summary
Resolves #731

- [x] Update institution labels
- [x] Make an optional digit in the phone number mask

## Relevant technical choices

- Refactor the mask utilities (`packages/web/utils/masks.js`)

## Checklist

- [ ] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [x] My code passes the linting.
- [ ] My code has proper inline documentation.
- [x] I have manually tested this PR.

### CODE REVIEW

File: packages/web/components/Form/MaskedInputField.js [hunk 1]
```diff
@@ -91,7 +92,7 @@ const MaskedInputField = ({
 				/>
 				{help && <Help id={name} label={label} HelpComponent={help} />}
 			</Row>
-			{errors && Object.keys(errors).length ? (
+			{Object.keys(errors)?.length ? (
```
Comment (luizeboli) [inline packages/web/components/Form/MaskedInputField.js:95]: hey man, there's a problem in that block

open your devtools and type:

```
const errors = undefined
Object.keys(errors)?.length
```

BOOM 💣 

`Cannot convert undefined or null to object`

That's why we need to check it

File: packages/web/utils/masks.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
+export const maskPatterns = {
```
Comment (luizeboli) [inline packages/web/utils/masks.js:1]: good boy 👍🏻

File: packages/web/utils/masks.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+		search: /(\d{5})(\d{3})/,
+		replace: '$1-$2',
+		pattern: /^\d{5}-\d{3}$/,
+		stringMask: '99999-999',
+		formatChars: null,
+	},
+};
+
+export const replaceWithMask = (value, pattern) => {
+	const { search, replace } = maskPatterns?.[pattern] || {};
```
Comment (luizeboli) [inline packages/web/utils/masks.js:36]: `maskPatterns` will always be there.

you can simple do `maskPatterns[pattern] || {}`

**Review by luizeboli — CHANGES_REQUESTED** [review]
pretty neat.

did some comments

File: packages/web/components/Form/MaskedInputField.js (hunk 1 above)
Comment (mateus4k) [reply packages/web/components/Form/MaskedInputField.js:95]: **Wow!**  🤯 🧠 

You rock, bro! 😎 

What do you think about doing this? 👇
```suggestion
			{Object.keys(errors || {})?.length ? (
```

Okay, I'm kidding, you know...

File: packages/web/utils/masks.js (hunk 3 above)
Comment (mateus4k) [reply packages/web/utils/masks.js:36]: Incredible! 🧠 🤯 🚀

File: packages/web/components/Form/MaskedInputField.js (hunk 1 above)
Comment (luizeboli) [reply packages/web/components/Form/MaskedInputField.js]: hahahaha sick!

**Review by luizeboli — APPROVED** [review]
@mateus4k did some small improvements 🎉

**Comment by mateus4k** [comment]
Tks, @luizeboli. You're awesome!

**Review by lcnogueira — APPROVED** [review]

**Pull request merged**